spec/lib/telephony/notification_sender_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'rails_helper'
+
+RSpec.describe Telephony::NotificationSender do
+  before do
+    Telephony::Test::Message.clear_messages
+    Telephony::Test::Call.clear_calls
+  end
+
+  context 'with the test adapter' do
+    subject do
+      described_class.new(
+        message: message,
+        to: to,
+        expiration: expiration,
+        channel: channel,
+        domain: domain,
+        country_code: country_code,
+        extra_metadata: { phone_fingerprint: 'abc123' },
+      )
+    end
+
+    let(:to) { '+1 (202) 262-1234' }
```

Comment:
i think we should remove the +1 here since that's the same as the country code

Reply:
I agree that's not clear, but it seems it was the way

Reply:
@svalexander, this is basically copy of the other class stated in the card. The country code seems redundant(not used), though it's part of the signature/interface.